cmd/oscap/main.go

```diff
@@ -233,7 +233,7 @@ func generateJson(dir, datastreamDistro, profileDescription, profile string) {
 	}
 	// hack to add an empty line at the end of the file for nicer diffs
 	bArray = append(bArray, '\n')
-	err = os.WriteFile(path.Join(dir, "customizations.json"), bArray, os.ModePerm)
+	err = os.WriteFile(path.Join(dir, "customizations.json"), bArray, 0640)
```

Comment:
Lets just disable the rule here? sounds bit too strict and seems it might be tricky to verify 0600 would work as well here.

Reply:
@croissanne can you comment? I am not sure what this does, shall I set 666 here?